### PR TITLE
teams/tidb: a vote for mjonss as the reviewer

### PR DESCRIPTION
**Pull request by bb7133**
As stated in the title.

Hi @pingcap/tidb-maintainers , PTAL.

### CODE REVIEW

**Review by jackysp — APPROVED** [review]

File: votes/0563-mjonss-as-tidb-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# A Vote for mjonss as TiDB Reviewer
+
+## Proposal
+
+[@mjonss](https://github.com/mjonss) has been working on `pingcap/tidb` for the past half year. His work covers several TiDB components(planner, executor, privileges and security, usability...). Also, he gave a lot of good suggestions in the review comments. I think @mjonss shows his qualification as a review of `pingcap/tidb` repository. The details of his contributions are listed as follow:
```
Comment (tisonkun) [inline votes/0563-mjonss-as-tidb-reviewer.md:5]:
```suggestion
[@mjonss](https://github.com/mjonss) has been working on `pingcap/tidb` for the past half year. His work covers several TiDB components(planner, executor, privileges and security, usability...). Also, he gave a lot of good suggestions in the review comments. I think @mjonss shows his qualification as a reviewer of `pingcap/tidb` repository. The details of his contributions are listed as follow:
```

**Review by tisonkun — COMMENTED** [review]
+1 (non-binding) @mjonss has done a lot of work on TiDB and easy to work with. Looking forward to further cooperation.

**Review by disksing — APPROVED** [review]

**Review by tiancaiamao — APPROVED** [review]

File: votes/0563-mjonss-as-tidb-reviewer.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+The vote will be open for at least 3 days unless there is an objection or not enough votes.
+
+## Scope
+
+* Team TiDB
+
+## Result
+
+(To be updated)Conclude the voting result, including approvals and vetoes, binding and non-binding.
```
Comment (tisonkun) [inline votes/0563-mjonss-as-tidb-reviewer.md:23]:
```suggestion
Approved by 3 binding votes and 1 non-binding vote:

* bb7133 (binding)
* tiancaiamao (binding)
* disksing (binding)
* tisonkun (non-binding)
```

**Comment by tisonkun** [comment]
@pingcap/tidb-maintainers it seems this voting already met the requirement. Could you help on concluding the result?

**Pull request merged**